subprojects/docs/src/samples/build-organization/standalone-precompiled-plugins/groovy/src/main/groovy/com.example.service.gradle

```diff
@@ -0,0 +1,34 @@
+// tag::plugins[]
+plugins {
+    id 'com.example.java-convention'
+}
+// end::plugins[]
+
+sourceSets {
+    integrationTest {
+        compileClasspath += sourceSets.main.output + sourceSets.test.output
+        runtimeClasspath += sourceSets.main.output + sourceSets.test.output
+    }
+}
+
+def integrationTest = tasks.register('integrationTest', Test) {
+    it.shouldRunAfter(test)
```

Comment:
💄 the `it.` could be omitted as the closure delegate is the task instance. This would look more like a DSL and be more inline with the Kotlin DSL version. Same in other configuration blocks.

Reply:
I know that it is implicit for Groovy, but IntelliJ does not recognize the methods otherwise and I can't navigate to them nor get autocompletion.

Reply:
Ah, that's unfortunate :/ Better leave it like this then 👍

Reply:
Well, everywhere else we omit the `it`. Should we change the rest of the documentation to start recommending `it` usage instead because IntelliJ?

Reply:
Fair point. I'll clean it up to be consistent with the rest of the docs.